query/src/main/scala/hmda/query/model/filing/Msa.scala

```diff
@@ -33,6 +16,40 @@ case class MsaWithName(
     homePurchase: Int,
     homeImprovement: Int,
     refinance: Int
+) {
+  def addName: MsaWithName = MsaWithName(
```

Comment:
In general, adding return types for methods is encouraged in typical `Scala` code.